README.md

```diff
@@ -131,7 +131,11 @@
 >>> primer1.E260
 248.40000000000006
 ```
-
+##### Nearest Neighbor
```

Comment:
Wrong place.
This section definitely needs to be placed under `#### Melting temperature`